### DISCUSSION TITLE
ccf-validation-tools/Bone-Marrow/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#259] [General]
# ccf-validation-tools/Bone-Marrow/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Bone-Marrow/comments/

**Comment by emquardokus** [comment]
Bone_Marrow_v1.4_DRAFT 
https://docs.google.com/spreadsheets/d/1z60ZA9r8Y435skSIjRwMX_EgBK0FN9up4CFfBQ0s944/edit#gid=771476671

8-30-2023
Added an ABBR column and put all author abbreviations for cell types there
**Different labels**
Corrected : Row 55 CD56 bright Natural Killer	CD16-negative, CD56-bright natural killer cell, human	CL:0000938
Corrected: Row 33 hemopoietic stem cell (HSC)	hematopoietic stem cell	CL:0000037

Mapped 3 cell types: 
ADDED Row34  lympho-myeloid progenitor cell	early lymphoid progenitor	CL:0000936	LMPP
ADDED Row 52 CD8 T cell effector memory	effector memory CD8-positive, alpha-beta T cell	CL:0000913
ADDED In row 54, no term id was found for the name/label mature CD8 T cell.
mature CD8 T cell	CD8-positive, alpha-beta T cell	CL:0000625

**This cell type does not seem to have a clear CL term:** @aleixpuigb need help with this one; not clear choice.
monocyte --> myeloid progenitor is stated in table:
In row 39, no term id was found for the name/label Myeloid progenitor (MOP); in literature see: https://doi.org/10.1182/blood-2019-128809

**Reply by aleixpuigb** [reply]
> monocyte --> myeloid progenitor is stated in table:
In row 39, no term id was found for the name/label Myeloid progenitor (MOP); in literature see: https://doi.org/10.1182/blood-2019-128809

It is strange that in the [article](https://doi.org/10.1182/blood-2019-128809) myeloid progenitor is not a progenitor of any other cell than monocytes. It is also strange that the abbreviation is MoP (which would make more sense to be Monocyte Progenitor, rather than Myeloid progenitor). However there is another `Monocyte progenitor` (MP).

I don't think that in CL there is a cell with those characteristics, and to be honest, from what I have been reading in literature, myeloid progenitor differentiate uniquely to monocytes. There are few monocytes progenitors (some are already in the table) that can be used, but none is a synonym of myeloid progenitor:

![image](https://github.com/hubmapconsortium/ccf-validation-tools/assets/94959119/f33800ed-c6e5-411b-b8af-1cad7acd40bd)

Looking at the table, I have realised that in some cases (e.g., monocyte --> myeloid progenitor) CT1 is the differentiated cell and CT2 there are the progenitor cells. On others the order is the expected, what makes it not consistent. This would also create a problem with the validation pipeline, since it will not find a valid relation (the differentiated cell will be the object instead the subject). I guess the reason is that the markers are only applicable to the last cell (CT2 in this case), but maybe adding multiple rows might be easier to understand and validate.
